app/views/samples/deletions/_destroy_multiple_confirmation_dialog.html.erb

```diff
@@ -37,11 +36,11 @@
         </div>
       </div>
 
-      <%= form_for(:deletion, url: destroy_multiple_namespace_project_samples_deletion_path,
+      <%= form_for(:deletion, url: destroy_samples_samples_deletions_path(namespace_id: @namespace.id),
```

Comment:
The route needs to be updated so we don't get the duplicate `_samples_samples_` in the path

Reply:
Hm I think we should still just have the one endpoint, and maybe we fire off a job if theres > 1 sample, otherwise delete directly? What are your thoughts? @ChrisHuynh333? @ericenns how about you?